src/omega_moderne_client/client/gpg_key_config.py

```diff
@@ -10,15 +11,28 @@ class GpgKeyConfig:
 
     @classmethod
     def load_from_env(cls) -> 'GpgKeyConfig':
+        env_vars = cls._load_env_variables(
+            "GPG_KEY_PASSPHRASE",
+            "GPG_KEY_PRIVATE_KEY",
+            "GPG_KEY_PUBLIC_KEY"
+        )
         return GpgKeyConfig(
-            key_passphrase=cls._load_env_variable("GPG_KEY_PASSPHRASE"),
-            key_private_key=cls._load_env_variable("GPG_KEY_PRIVATE_KEY"),
-            key_public_key=cls._load_env_variable("GPG_KEY_PUBLIC_KEY")
+            key_passphrase=env_vars[0],
```

Comment:
Why did you move from environment variable to variable location based on argument being passed?